src/components/TodoCount.js

```diff
@@ -0,0 +1,13 @@
+const TodoCount = ({$todoCount, currentState, currentFilterState}) => {
+    let todoCount = 0;
+    if(currentFilterState === "all") {
+        todoCount = currentState.length
+    } else if (currentFilterState === "active") {
+        todoCount = currentState.filter(todo => todo.isCompleted === false).length
+    } else {
+        todoCount = currentState.filter(todo => todo.isCompleted === true).length
+    }
+    $todoCount.innerHTML = `총 <strong>${todoCount}</strong> 개`;
```

Comment:
범위를 좁히면 `$todoCount.innerText`로 바로 카운트에 접근 가능해요.
```suggestion
    $todoCount.innerText = todoCount;
```
innerHTML은 HTML을 모두 파싱하기 때문에 innerText, textContent 보다 비교적 성능이 떨어진다네요!

Reply:
헉s 개꿀팁이넹.. 감사합니다!
사실 저 생각을 못했어요 `strong` 태그에 저렇게 접근할 수 있다는 생각을 아예 못해서 저렇게 구현을 한거거든요!

진짜 개꿀팁이네요! 적용해보겠습니다 😎